app/helpers/web/admin/categories_helper.rb

```diff
@@ -0,0 +1,42 @@
+module Web::Admin::CategoriesHelper
+  def get_categories_tree
+    return if @categories_tree == [{}]
+    (content_tag(:ul, class: '') do
+      @categories_tree.each do |root|
+        concat(content_tag(:li, class: '') do
+          concat(get_categories_tree_by_category(root))
+        end)
+      end
+    end)
+  end
+
+  def get_categories_tree_by_category(category)
+    if category[:childs].present?
+      concat link_to(category[:category_name], "#")
+      concat content_tags_for_edit_category(category[:id])
+      (content_tag(:ul, class: '') do
+        category[:childs].each do |child|
+          concat(content_tag(:li, class: '') do
+            concat(get_categories_tree_by_category(child))
+          end)
+        end
+      end)
+    else
+      concat link_to(category[:category_name], '#')
+      content_tags_for_edit_category(category[:id])
+    end
+  end
+
+  def content_tags_for_edit_category(href)
+    tags = (link_to(new_admin_category_path(:id => href.to_s), class: 'btn btn-xs btn-success')do
```

Comment:
Line is too long. [97/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.